src/helpers/skipForPrerelease.ts

```diff
@@ -0,0 +1,15 @@
+import type { BranchObject } from 'semantic-release';
+import { PluginConfigType } from '../types';
+
+const isPrerelease = (branchObject: BranchObject) => !!branchObject?.prerelease;
+
+const skipForPrerelease = (
```

Comment:
The concept of skipping, with double negation (ignore skipping) seems awkward. How do you feel about a name like `includePrerelease`? I wonder if that clarity of name might make it easier to spot logical bugs in the implementation

But that's a minor nit, this code appears to be functional to me!

Reply:
Very good point, thanks. I'll think about this later today, the double negation seems logical in some way, but at the same time pretty confusing hah.

Reply:
Changed it up a little bit and now it's `includePrerelease` instead of `ignoreSkipForPrerelease`

Reply:
Of course! 

Looks great :rocket: